backend/keystore/keystore.go

```diff
@@ -38,6 +38,7 @@ const (
 var ErrSigningAborted = errors.New("signing aborted by user")
 
 // Keystore supports hardened key derivation according to BIP32 and signing of transactions.
+//go:generate moq -pkg mocks -out mocks/keystore.go . Keystore
```

Comment:
before you say anything @x1ddos, this tool is the one we looked at that produces mocks nearly like I'd write them by hand :D

Reply:
I remember :)
I still don't like the separate package `mocks`. It's often more useful in the same package as that one under test. plus, all those 10+ ugly sync.RWMutex.